src/async-wrap.cc

```diff
@@ -164,8 +168,6 @@ static void DestroyIdsCb(uv_idle_t* handle) {
       FatalException(env->isolate(), try_catch);
     }
   }
-
-  env->destroy_ids_list()->clear();
```

Comment:
@addaleax If you don't clear the `std::vector<double>` then destroy ids list will continue to call the destroy hooks for all the previously entered asyncIds.